index.html

```diff
@@ -52,6 +52,195 @@ <h4 class="social-title">LET'S CONNECT</h4>
           </li>
         </ul>
       </section>
+
+      <div class="works">
+        <div class="card">
+          <header class="card">
+            <img
+              src="./images/snapshot1.png"
+              class="card-header-pic"
+              alt="Projects Header"
+            />
+          </header>
```

Comment:
- [ ] It is recommended to use CSS background-image property for this part instead of `<img>` tag. Also `<header>` tag is only semantically correct when used on the navigation bar not within the page.

Reply:
Done!